docs/book/src/tasks/experimental-features/cluster-class/change-clusterclass.md

```diff
@@ -174,17 +174,15 @@ underlying objects like control plane and MachineDeployment act in the same way
 The topology reconciler enforces values defined in the ClusterClass templates into the topology
 owned objects in a Cluster.
 
-A simple way to understand this is to `kubectl get -o json` templates referenced in a ClusterClass;
-then you can consider the topology reconciler to be authoritative on all the values
-under `spec`. Being authoritative means that the user cannot manually change those values in
-the object derived from the template in a specific Cluster (and if they do so the value gets reconciled
-to the value defined in the ClusterClass).
+More specifically, the topology controller uses [Server Side Apply](https://kubernetes.io/docs/reference/using-api/server-side-apply/)
+to write/patch topology owned objects; using SSA allows other controllers to co-author the generated objects, 
+like e.g. integration info for the subnet list in CAPA.
 
 <aside class="note">
 <h1>What about patches?</h1>
 
 The considerations above apply also when using patches, the only difference being that the
-authoritative fields should be determined by applying patches on top of the `kubectl get -o json` output. 
+authoritative fields should be determined by applying patches on top of the templates. 
```

Comment:
The reference to authoritative fields now sounds a bit strange as we removed the text it was referring to.

Maybe just shorten to:

> The considerations above apply also when using patches.

Reply:
Similarly below we are talking about non-authoritative. Not sure if it is still clear what that means

Reply:
re-written, PTAL